pkg/kubectl/kubectl.go

```diff
@@ -49,7 +109,7 @@ func GetVersionInfo(c Client) (string, string, error) {
 			}
 			return "", "", fmt.Errorf("kubectl error (%v): %s", err, stderr)
 		}
-		return "", "", fmt.Errorf("error parsing kubectl output: %v", parseErr)
+		return "", "", fmt.Errorf("error parsing kubectl version output: %s", parseErr)
```

Comment:
Outside the scope of this PR, but would be great if we could validate and agree on a minimum required cluster and `kubectl` version that we support.